models/engine/db_storage.py

```diff
@@ -74,3 +74,30 @@ def reload(self):
     def close(self):
         """call remove() method on the private session attribute"""
         self.__session.remove()
+
+    def get(self, cls, id):
+        """retrieves one object"""
+        try:
```

Comment:
It's probably better to check if the all() method returns an empty list vs this general try and except statement. The try and except statement used here can potentially mask unexpected errors that may lead to hard-to-find bugs.